tests/test_discriminator.py

```diff
@@ -45,14 +45,12 @@ def test_serialize(self):
         dtm.value = StringValue(name='foo', value='Hello')
         dtm.values = [NumberValue(name='bar', value=5), RenamedValue(name='baz', value='World')]
         assert dtm._serialize() == {
```

Comment:
Should we call out that the format is changing? I know at least one piece of code which it'll disrupt (luckily I could increase the cache version there along with the PynamoDB update).

Reply:
I'll put a release note in -- I was also planning in the JSON pr to make the serialize/deserialize methods public.